src/shared/views/Mapper/Tile.js

```diff
@@ -151,15 +168,23 @@ export class _Tile extends React.Component<Props> {
                         borderColor: 'rgba(255,255,255,0.2)',
                     }}
                     source={imageSource}
-                />
+                >
+                    <ImageBackground
```

Comment:
It's a bit counter intuitive to have ImageBackgorund twice, since one of it is the overlay. But this is probably a react-native thing...

Reply:
yes, they need to be backgrounds, because we have the `Touchable` element in front of it, which responds to taps. At least it's my understanding!